app/src/main/java/com/a1ishka/bankaccount/presentation/account/AccountViewModel.kt

```diff
@@ -0,0 +1,60 @@
+package com.a1ishka.bankaccount.presentation.account
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.a1ishka.bankaccount.data.entity.AccountEntity
+import com.a1ishka.bankaccount.domain.AccountDao
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+class AccountViewModel @Inject constructor(
+    private val accountDao: AccountDao
```

Comment:
ViewModel use repository or use cases to communicate with DB. Read about Room structure in documentation

Reply:
replaced by repository